### DISCUSSION TITLE
Few questions encountered using GPGPU-Sim

### DISCUSSION THREAD

**Discussion by jonnyhq** [accel-sim/accel-sim-framework#227] [General]
Hi all,
I am a new user about GPU simulator. I have a few questions and want to ask you for help. Thank you in advance.
1. There are many benchmarks in the website<https://github.com/gpgpu-sim/gpgpu-sim_simulations>, some of them can't be run maybe because of my computer configuration, then the runtime of others is too short. So I want to know whether exist some benchmarks run in a long time, and where can I find them?
2. I ever tried to modify the frequency of core clock, interconnect clock, l2 clock, dram clock in the file "gpgpusim.config". Like this :-gpgpu_clock_domains 2000.0:2000.0:2000.0:924.0
Then according to document obtained <gpgpusim_power_trace_report.log.gz>, I find that power of sum or component power are not increased when increasing the frequency and even little decreased. Why is it ? When the frequency increased, the GPU should consume more power. So do you know what is happened?
3. While the GPU is running in the simulator, can I change the frequency, such as core clock etc., at some point in time?
Looking forward to you reply quickly, thank you!

A new user.

**Comment by JRPan** [comment]
1. You can change parameters to make it longer. Or you can write your own ones. 
2. If you are using gpuwattach, please ask them as we don't use that anymore. But gpgpusim simulate cycles. Higher clock rate consumes more power, but the duration is shorter. 
3. No. But you can implement that yourself.

**Reply by jonnyhq** [reply]
Thanks for your answer! There are still some issues.
1. What parameters can I change? 
2. About gpuwattch, where can I find them?
3. Do you mean that I can not change the frequency after some time step, and the whole process must run in the same frequency on the current basis？

**Reply by JRPan** [reply]
1. Depends on what you are trying to do. Some benchmarks depend on the size of inputs. For some others, you can go into the code and change how many threads are issued or how many iterations a loop goes to make it run longer. 
2. http://gpgpu-sim.org/gpuwattch/ and ask folks at https://github.com/gpgpu-sim/gpgpu-sim_distribution
3. Correct. The options are parsed in the beginning of the simulation. I don't know how much it matters to the power tho. I don't know if gpuwattach even supports DVFS. If you are looking for a modern power GPU model, you can check out Accel-Wattach https://github.com/accel-sim/accel-sim-framework/blob/release/AccelWattch.md. 

Thanks
